examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java

```diff
@@ -479,8 +477,7 @@ public static void main(String[] args) throws IOException {
     if (options.getOutputToDatastore()) {
       toWrite
       .apply(ParDo.named("FormatForDatastore").of(new FormatForDatastore(options.getKind())))
-      .apply(DatastoreIO.writeTo(MoreObjects.firstNonNull(
-          options.getOutputDataset(), options.getProject())));
+      .apply(DatastoreIO.writeTo(options.getProject()));
```

Comment:
please keep the OutputDataset option here. You may rename it to OutputProject if you like.